src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,116 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+interface Props {
+  onAdd: (
+    movie: {
+      title: string,
+      description: string,
+      imgUrl: string,
+      imdbUrl: string,
+      imdbId: string,
+    }
+  ) => void
```

Comment:
you have a type for movie
```suggestion
  onAdd: (
    movie: (movie: Movie) => void
```

Reply:
isn't fixed